buildSrc/src/main/kotlin/Config.kt

```diff
@@ -11,7 +11,7 @@ object MavenUrl {
 
 object Version {
   const val bintray = "1.8.4"
-  const val kotlin = "1.3.10"
```

Comment:
This is required for Gradle 6.1.1 running with Kotlin DSL

Reply:
Why don't we bump to the latest? It looks like new version is innocuous.